CSS_files/components/cards/cards.css

```diff
@@ -0,0 +1,121 @@
+
+.card-content{
+    display: flex;
+    flex-direction: column;
+    
+}
+
+.card img{
```

Comment:
You can use a class named as card-img instead of selecting the element inside the card and pass the properties required inside it.

Reply:
Got it, Roshan. Thank you.